src/styles/theme.js

```diff
@@ -147,6 +147,10 @@ export default ({ color, backgroundColor, borderColor, cardColor }: Props) => ({
   heading2: {
     fontSize: 20,
   },
+  headingLarge: {
+    fontSize: 40,
+    color: 'rgb(100, 100, 100)',
```

Comment:
Hardcoding the color like this would break the view in night mode, so it's preferably better to take the font color from styles.js